src/main/java/com/terraformersmc/terrestria/init/TerrestriaConfiguredFeatures.java

```diff
@@ -68,128 +68,138 @@
-				new SimpleBlockStateProvider(Blocks.OAK_LEAVES.getDefaultState()),
-				new SmallLogSphereFoliagePlacer(UniformIntDistribution.of(1), UniformIntDistribution.of(0)),
 				new SpindlyTrunkPlacer(10, 0, 0),
+				new SimpleBlockStateProvider(Blocks.OAK_LEAVES.getDefaultState()),
+				new SimpleBlockStateProvider(Blocks.OAK_SAPLING.getDefaultState()),
+				new SmallLogSphereFoliagePlacer(ConstantIntProvider.create(1), ConstantIntProvider.create(0)),
 				new TwoLayersFeatureSize(1, 0, 0))
 				.build());
 
 		JUNGLE_PALM_TREE = registerSandyTree("jungle_palm_tree", new TreeFeatureConfig.Builder(
 				new SimpleBlockStateProvider(Blocks.JUNGLE_WOOD.getDefaultState()),
-				new SimpleBlockStateProvider(TerrestriaBlocks.JUNGLE_PALM_LEAVES.getDefaultState()),
-				new PalmFanFoliagePlacer(UniformIntDistribution.of(3), UniformIntDistribution.of(0)),
 				new BentTrunkPlacer(15, 15, 15),
+				new SimpleBlockStateProvider(TerrestriaBlocks.JUNGLE_PALM_LEAVES.getDefaultState()),
+				new SimpleBlockStateProvider(TerrestriaBlocks.JUNGLE_PALM_SAPLING.getDefaultState()),
+				new PalmFanFoliagePlacer(ConstantIntProvider.create(3), ConstantIntProvider.create(0)),
 				new TwoLayersFeatureSize(1, 0, 2))
 				.ignoreVines()
 				.build());
 
-		SMALL_HEMLOCK_TREE = registerTree("small_hemlock_tree", spruceOf(TerrestriaBlocks.HEMLOCK));
-		SMALL_REDWOOD_TREE = registerTree("small_redwood_tree", spruceOf(TerrestriaBlocks.REDWOOD));
+		SMALL_HEMLOCK_TREE = registerTree("small_hemlock_tree", spruceOf(TerrestriaBlocks.HEMLOCK, TerrestriaBlocks.HEMLOCK_SAPLING.getDefaultState()));
+		SMALL_REDWOOD_TREE = registerTree("small_redwood_tree", spruceOf(TerrestriaBlocks.REDWOOD, TerrestriaBlocks.REDWOOD_SAPLING.getDefaultState()));
 
-		HEMLOCK_TREE = registerTree("hemlock_tree", tallSpruceOf(TerrestriaBlocks.HEMLOCK, 24, 4, 3, 2, 5, 1, 11));
-		REDWOOD_TREE = registerTree("redwood_tree", tallSpruceOf(TerrestriaBlocks.REDWOOD, 24, 4, 3, 5, 2, 12, 7));
+		HEMLOCK_TREE = registerTree("hemlock_tree", tallSpruceOf(TerrestriaBlocks.HEMLOCK, TerrestriaBlocks.HEMLOCK_SAPLING.getDefaultState(), 24, 4, 3, 2, 5, 1, 11));
+		REDWOOD_TREE = registerTree("redwood_tree", tallSpruceOf(TerrestriaBlocks.REDWOOD, TerrestriaBlocks.REDWOOD_SAPLING.getDefaultState(), 24, 4, 3, 2, 5, 7, 12));
 
-		MEGA_HEMLOCK_TREE = registerQuarteredMegaTree("mega_hemlock_tree", giantSpruceOf(TerrestriaBlocks.HEMLOCK, 32, 8, 7, 2, 5, 1, 11));
-		MEGA_REDWOOD_TREE = registerQuarteredMegaTree("mega_redwood_tree", giantSpruceOf(TerrestriaBlocks.REDWOOD, 32, 8, 7, 2, 5, 12, 7));
+		MEGA_HEMLOCK_TREE = registerQuarteredMegaTree("mega_hemlock_tree", giantSpruceOf(TerrestriaBlocks.HEMLOCK, TerrestriaBlocks.HEMLOCK_SAPLING.getDefaultState(), 32, 8, 7, 2, 5, 1, 11));
+		MEGA_REDWOOD_TREE = registerQuarteredMegaTree("mega_redwood_tree", giantSpruceOf(TerrestriaBlocks.REDWOOD, TerrestriaBlocks.REDWOOD_SAPLING.getDefaultState(), 32, 8, 7, 2, 5, 7, 12));
```

Comment:
Why did these values get shuffled around?

Reply:
The numbers stand for 
```java
int height, int randomHeight, int extraRandomHeight, int baseRadius, int randomRadius, int baseBareHeight, int randomBareHeight
```
Since I'm no Terrestria dev I don't really know if this change is correct or not; can someone tell me if this is intended? @NeusFear @coderbot16

Reply:
I can't see a reason that they were changed. I guess just check the way the old and new ones look an see if there is a difference?

Reply:
@haykam821 Do you remember the reason why you changed it?

Reply:
Until you answer, I'm going to revert this change now to be consistent with the 1.16 release